react-hash-routing/client/components/header.jsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+
+export default function Header(props) {
+  return (
+    <header className="mb-5">
+      <nav className="navbar navbar-dark bg-dark shadow-sm">
+        <div className="container">
+          <div className="col px-0">
+            {/* this anchor should go back to the catalog at '#' */}
+            <a href="" className="navbar-brand">
+              <i className="fa fa-dollar-sign" /> Wicked Sales
```

Comment:
There is one here too.

Reply:
<img width="595" alt="image" src="https://user-images.githubusercontent.com/99709530/171034875-6eeee7e1-5850-4228-b009-1558c6ee11e1.png">